### ISSUE TITLE
Problem with traffic signs

### ISSUE THREAD

**Issue by DaddyWesker** [chenhsuanlin/inverse-compositional-STN#15]
Hello.
Just tried to run your code. Mnist-tensorflow works fine, but traffic signs shows some strange results... Can you tell me, if just running train.py with option, for example IC-STN, is enough in case of signs?

![problem_with_signs](https://user-images.githubusercontent.com/10774222/52549812-29e34900-2de6-11e9-9864-ca64b5e078db.png)

here is what i've got.

**Comment by chenhsuanlin** [comment]
It seems that I set the default `opt.lrGP` to be too high for IC-STN. I tried running with the learning rate set to be 3e-5 and it seems to be able to train through 1M iterations. This has been updated, sorry about this!
